mkosi

```diff
@@ -2287,7 +2287,7 @@ def load_args():
             if platform.machine() == "aarch64":
                 args.mirror = "http://mirror.archlinuxarm.org"
         elif args.distribution == Distribution.opensuse:
-            args.mirror = "https://download.opensuse.org"
+            args.mirror = "http://download.opensuse.org"
```

Comment:
Running `zypper --root /tmp/mkosi-zsj8e3kx/root addrepo -ck https://download.opensuse.org/tumbleweed/repo/oss/ Main` here (TW) works fine.

So there might be something wrong with your setup.

Could you try to run the same command and if it indeed fails then ask some help to the openSUSE community ?